### PR TITLE
add a left_eigenvectors() method

### PR DESCRIPTION
**Pull request by certik**
This is a rebase and polished up version of #8721.

I took the diff from @zingale, created a simple commit out of it with his name, and then added few patches of my own polishing up the docstring.

### CODE REVIEW

File: sympy/matrices/matrices.py [hunk 1]
```diff
@@ -3084,6 +3084,44 @@ def eigenvects(self, **flags):
                 out.append((r, k, [mat._new(b) for b in basis]))
         return out
 
+    def left_eigenvects(self, **flags):
+        """Returns left eigenvectors and eigenvalues.
+
+        This function returns the list of triples (eigenval, multiplicity,
+        basis) for the left eigenvectors. Options are the same as for
+        eigenvects(), i.e. the **flags arguments gets passed directly to
```
Comment (asmeurer) [inline sympy/matrices/matrices.py]: Sphinx does something weird with `**flags`, so you should put it in double backquotes.

**Comment by asmeurer** [comment]
There is an error when building the docs:

```
Warning, treated as error:
/Users/aaronmeurer/Documents/python/sympy/sympy/sympy/matrices/matrices.py:docstring of sympy.matrices.matrices.MatrixBase.is_square:18: WARNING: Inline strong start-string without end-string.
```

It could be related to that `**` thing.

**Comment by asmeurer** [comment]
Other than that comment I am +1.

File: sympy/matrices/matrices.py (hunk 1 above)
Comment (certik) [inline sympy/matrices/matrices.py]: Good point, I just push in a patch that does that.

**Comment by certik** [comment]
Tests pass, merging.

**Comment by zingale** [comment]
Thank you all for helping (sorry I've been quiet -- just getting back to things after travel.

**Pull request merged**